reboot-core/src/test/kotlin/nl/thanus/reboot/ReBootKtTest.kt

```diff
@@ -0,0 +1,34 @@
+package nl.thanus.reboot
+
+import com.github.ajalt.clikt.core.BadParameterValue
+import com.github.ajalt.clikt.core.MissingParameter
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertDoesNotThrow
+import org.junit.jupiter.api.assertThrows
+
+internal class ReBootKtTest {
+
+    @Test
+    fun `Directory should always be specified in the CLI`() {
+        val emptyCliArgs = arrayOf<String>()
+        assertThrows<MissingParameter> { ReBoot().parse(emptyCliArgs) }
+    }
+
+    @Test
+    fun `Excluding refactorings is optional`() {
+        val cliArgs = arrayOf("directory")
+        assertDoesNotThrow { ReBoot().parse(cliArgs) }
+    }
+
+    @Test
+    fun `Excluding refactorings with flag -e and --excluded is accepted`() {
+        val cliArgs = arrayOf("differentDirectory", "-e", "request-mappings", "--excluded", "autowired-field-injection")
```

Comment:
Perhaps reuse the enum in excluded arguments?
```suggestion
        val cliArgs = arrayOf(
                "differentDirectory",
                "-e", Refactoring.REQUEST_MAPPINGS.refactoring,
                "--excluded", Refactoring.AUTOWIRED_FIELD_INJECTION.refactoring
        )
```